base/local/thanos-rule.yaml

```diff
@@ -116,23 +105,6 @@ spec:
             memory: 200Mi
           limits:
             memory: 500Mi
-      - name: config-watch
```

Comment:
why not keep this as base?

Reply:
configmap generator will do the work for this

Reply:
> The ConfigMap declared from a ConfigMapGenerator is treated differently. A hash is appended to the name and any change in the ConfigMap will trigger a rolling update.

Reply:
ah, I see, thanks!